### DISCUSSION TITLE
vscode copilot extension only prompts to connect to personal github, not github enterprise

### DISCUSSION THREAD

**Discussion by jamesgriggs** [community/community#68015] [Enterprise]
### Select Topic Area

Bug

### Body

I am logged into GHE on vscode, but GH copilot is prompting me to sign in with github (not my enterprise URL). I cannot get it to try to log in with enterprise. 

Images below showing my being logged in and my settings. GHE connection worked perfectly easily with GH Pull Requests extension

![image](https://github.com/community/community/assets/32936211/ffd8164f-14da-4aad-b393-e28416f88439)

![image](https://github.com/community/community/assets/32936211/88a41f15-a426-4b1a-b57a-38708bf5949f)

**Comment by Akash1134** [comment]
Hi @jamesgriggs ,

Sorry about the issue with signing in using your enterprise account. Have you tried signing out of both the accounts, your IDE, and then check for any pending updates for VSCode and Copilot?

Thanks!
